### PR TITLE
Feature/title

### PR DESCRIPTION
**Pull request by Khanjan04**
Extra text removed and Title added in challenges and mine page.

### CODE REVIEW

**Comment by Khanjan04** [comment]
Done. Please have a look.

**Comment by Khanjan04** [comment]
Issue No #19

File: src/pages/challenges.tsx [hunk 1]
```diff
@@ -2,15 +2,18 @@ import { FunctionComponent } from 'react';
...
+import Title from 'components/Title';
 import ChallengesJson from 'mocks/challenges.json';
 import { CHALLENGES_SCREEN } from '../components/constants/display-sections.js';
+import { Helmet } from 'react-helmet';
 
 const Challenges: FunctionComponent = () => {
   return (
     <Layout>
+      <Helmet>
+        <title>Challenges</title>
```
Comment (whyDontI) [inline src/pages/challenges.tsx:14]: Can you make it `Challenges | Status Real Dev Squad` for all the pages

File: src/pages/mine.tsx [hunk 2]
```diff
@@ -2,12 +2,15 @@ import { FunctionComponent } from 'react';
 import Layout from '../components/Layout';
 import Navbar from '../components/navbar';
 import Title from '../components/Title';
+import { Helmet } from 'react-helmet';
 
 const Mine: FunctionComponent = () => {
   return (
     <Layout>
+      <Helmet>
+        <title>Mine</title>
```
Comment (whyDontI) [inline src/pages/mine.tsx:11]: Can you make it `Mine | Status Real Dev Squad`?

**Review by whyDontI — CHANGES_REQUESTED** [review]
Great work 👍🏽 
Requested some changes.

**Comment by whyDontI** [comment]
Can you also add the title for the `index` page?

**Comment by Khanjan04** [comment]
Let me do that

**Comment by Khanjan04** [comment]
> Can you also add the title for the `index` page?

Index page? You mean Task page

**Comment by Khanjan04** [comment]
Please have a look now.

**Review by whyDontI — APPROVED** [review]
Great work @Khanjan04 👍🏽 🥳

**Pull request merged**